sdk/cognitivelanguage/ai-language-text/test/public/analyzeBatch.spec.ts

```diff
@@ -861,6 +862,33 @@ matrix([["APIKey", "AAD"]] as const, async (authMethod: AuthMethod) => {
           await assertActionsResults(await poller.pollUntilDone(), expectation15);
         });
 
+        it("whole batch input with auto language detection", async function () {
+          const docs = [
+            "I will go to the park.",
+            "Este es un document escrito en Español.",
+            "猫は幸せ",
+          ];
+          const poller = await client.beginAnalyzeBatch(
+            [
+              {
+                kind: AnalyzeBatchActionNames.EntityRecognition,
+              },
+              {
+                kind: AnalyzeBatchActionNames.PiiEntityRecognition,
+              },
+              {
+                kind: AnalyzeBatchActionNames.KeyPhraseExtraction,
+              },
```

Comment:
We could probably get away with just one action for this test
```suggestion
```

Reply:
But on the other hand, I believe some other actions has issues with representing the output property correctly. Could we update this instead to have a list of all supported actions?